<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>415</x>
          <y>-1</y>
          <width>157</width>
          <height>49</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="horizontalSpacing">
          <number>4</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="toolButtonPatronymic">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBoxPatronymic"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>157</x>
          <y>120</y>
          <width>411</width>
          <height>46</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_12">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <widget class="QToolButton" name="toolButtonAdd">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="toolButtonFind">
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="toolButtonDelete">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QToolButton" name="toolButtonSave">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButtonRefresh">
           <property name="text">
            <string>Обновить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QToolButton" name="toolButtonPrint">
           <property name="text">
            <string>Печать</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>60</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="horizontalSpacing">
          <number>20</number>
         </property>
         <property name="verticalSpacing">
          <number>-1</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_9">
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Дом</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEditBuilding"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_10">
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Корпус</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEditCorpus"/>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QGridLayout" name="gridLayout_11">
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>!Квартира</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEditApartment">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>110</y>
          <width>127</width>
          <height>47</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>5</number>
         </property>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEditPhone"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Телефон</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>581</width>
          <height>192</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>161</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxSurname">
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::InsertAtBottom</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToContents</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonSurname">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>59</y>
          <width>160</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Улица</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxStreet"/>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonStreet">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>0</y>
          <width>160</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxName">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonName">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Phone Book</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSurname"/>
   <addaction name="actionName"/>
   <addaction name="actionPatronymic"/>
   <addaction name="actionStreet"/>
  </widget>
  <action name="actionSurname">
   <property name="text">
    <string>Surname</string>
   </property>
   <property name="toolTip">
    <string>goto Surname Dictionary</string>
   </property>
  </action>
  <action name="actionName">
   <property name="text">
    <string>Name</string>
   </property>
   <property name="toolTip">
    <string>goto Name dictionary</string>
   </property>
  </action>
  <action name="actionPatronymic">
   <property name="text">
    <string>Patronymic</string>
   </property>
   <property name="toolTip">
    <string>goto Patronimyc dictionary</string>
   </property>
  </action>
  <action name="actionStreet">
   <property name="text">
    <string>Street</string>
   </property>
   <property name="toolTip">
    <string>goto Street</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>toolButtonPatronymic</tabstop>
  <tabstop>toolButtonStreet</tabstop>
  <tabstop>lineEditBuilding</tabstop>
  <tabstop>lineEditCorpus</tabstop>
  <tabstop>lineEditApartment</tabstop>
  <tabstop>lineEditPhone</tabstop>
  <tabstop>toolButtonFind</tabstop>
  <tabstop>toolButtonAdd</tabstop>
  <tabstop>toolButtonDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
